tests/integration/tests.py

```diff
@@ -48,36 +51,376 @@ def launch(self, env_vars=None):
             check=True)
 
 
-def test_gpt_neo():
-    with Runner('deepspeed') as r:
-        prepare.build_hf_handler_model("gpt-neo-2.7b")
-        r.launch()
-        client.run("huggingface gpt-neo-2.7b".split())
+class TestHfHandler:
+    # Runs on g5.12xl
```

Comment:
Do we want to actually specify the instance, or should we just note the max accelerator count (max TP degree) for the test class? Reason I ask is when we move up to g6, do we want to need to update these with every instance change per se but do want the comment to help us debug issues when we do change an instance.

`# Runs on GPU - max TP=4` or something like that

Reply:
For now, this is just a comment to help keep track of things. The plan is to eventually change this into a pytest mark so we could do something like run all tests with mark `gpu-4`. I'll fix this when I start adding the marks